.github/workflows/build-and-test.yml

```
@@ -1,25 +1,52 @@
+    - "main"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true      
 
 jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+
+    steps: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)